src/config/ConfigManager.cpp

```diff
@@ -818,6 +819,8 @@ void CConfigManager::postConfigReload(const Hyprlang::CParseResult& result) {
     if (Debug::disableStdout && isFirstLaunch)
         Debug::log(LOG, "Disabling stdout logs! Check the log for further logs.");
 
+    Debug::coloredLogs = std::any_cast<Hyprlang::INT>(m_pConfig->getConfigValue("debug:colored_stdout_logs"));
```

Comment:
this will make it unchangeable dynamically. Why not do the same as with `debug:disable_logs`?

Reply:
i think its the opposite. this is on config reload so it is changeable dynamically. if i put it next to disable_logs it only gets run once and that will be unchangeable

Reply:
dynamically means with hyprctl.

`disable_logs` is a static pointer, not a value copy.

Reply:
changed